tomviz/PythonUtilities.h

```diff
@@ -178,6 +185,9 @@ class Python
   /// vtkPythonPythonInterpreter::PrependPythonPath(...)  to do the work.
   static void prependPythonPath(std::string dir);
 
+  /// Create a threaded Dataset object for operators to use
```

Comment:
I think the term 'threaded' is a little overloaded, may be we should  stick with 'internal pipeline' and 'external pipeline'?

Reply:
Fixed.